src/chat/runeMagicCard.ts

```diff
@@ -0,0 +1,360 @@
+import { ItemDataSource } from "@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/data/data.mjs/itemData";
+import { Rune } from "../actors/item-specific/rune";
+import { RqgActor } from "../actors/rqgActor";
+import { RqgActorDataSource } from "../data-model/actor-data/rqgActorData";
+import { CultDataSource } from "../data-model/item-data/cultData";
+import { ItemTypeEnum, RqgItemDataSource } from "../data-model/item-data/itemTypes";
+import { RuneDataSource } from "../data-model/item-data/runeData";
+import { Ability, ResultEnum, ResultMessage } from "../data-model/shared/ability";
+import {
+  assertItemType,
+  getActorFromIds,
+  getGame,
+  getJournalEntryName,
+  getSpeakerName,
+  RqgError,
+  usersThatOwnActor,
+} from "../system/util";
+
+type ShortRune = {
+  name: string;
+  id: string;
+  chance: number;
+};
+
+type RuneMagicCardFlags = {
+  actorId: string;
+  tokenId: string | null; // Needed to avoid saving a full (possibly syntetic) actor
+  itemData: ItemDataSource;
+  eligibleRunes: RuneDataSource[];
+  formData: {
+    runePointCost: number;
+    cultId: string;
+    magicPointBoost: number;
+    ritualOrMeditation: number;
+    skillAugmentation: number;
+    otherModifiers: number;
+    chance: number;
+    selectedRuneId: string;
+    journalEntryName: string;
+    journalPack: string;
+    ritualOrMeditationOptions: any;
+    skillAugmentationOptions: any;
+  };
+};
+
+export class RuneMagicCard {
+  public static async show(
+    runeMagicItemId: string,
+    actor: RqgActor,
+    token: TokenDocument | null
+  ): Promise<void> {
+    const runeMagicItem = actor.items.get(runeMagicItemId);
+    assertItemType(runeMagicItem?.data.type, ItemTypeEnum.RuneMagic);
+    const cult = actor.items.get(runeMagicItem.data.data.cultId);
+    assertItemType(cult?.data.type, ItemTypeEnum.Cult);
+
+    let usableRuneNames: string[] = [];
+    let runesForCasting: RuneDataSource[] = [];
+    if (runeMagicItem.data.data.runes.includes("Magic (condition)")) {
```

Comment:
I don't have any solution and I do the same thing somewhere else, but hardcoding the rune name makes it hard to translate. I have special skill names as [config](https://github.com/sun-dragon-cult/fvtt-system-rqg/blob/160aae27f89fd51e1a56a572f72dc206cda8a03b/src/system/config.ts#L22) - could this special rune possibly be handled the same way?

Reply:
Made the "magicRuneName" setting and used it.

Reply:
That's much better than the config I have for the skills. I'll add a case for refactoring the special skills into a setting as well. I guess all of those could become a FormApplication for setting "special" items names. It possibly ties in a little with #123 as well (even though I don't want all rune names to be settings)